external/mbedtls/alt/aes_alt.c

```diff
@@ -201,12 +201,7 @@ int mbedtls_aes_crypt_cfb128(mbedtls_aes_context *ctx,
 	hal_aes_param aes_param = {HAL_AES_CFB128, iv, 16, iv_off, NULL, NULL, NULL};
 	int ret = 0;
 
-	if (mode == MBEDTLS_AES_ENCRYPT) {
-		ret = sl_aes_encrypt(ctx->shnd, &aes_in, &aes_param, ctx->enc_key_idx, &aes_out);
-	} else if (mode == MBEDTLS_AES_DECRYPT) {
-		ret = sl_aes_decrypt(ctx->shnd, &aes_in, &aes_param, ctx->dec_key_idx, &aes_out);
-	}
-
+	ret = sl_aes_encrypt(ctx->shnd, &aes_in, &aes_param, ctx->enc_key_idx, &aes_out);
```

Comment:
Why don't you remove 'mode' param if this is not necessary anymore?

Reply:
Thanks for review. I found that software method of mbedtls_aes_cyprt_cfb128 manipulate input and output depends on mode. I'll modify it.